v2/spanner-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/dbutils/dml/MySQLDMLGenerator.java

```diff
@@ -389,7 +389,9 @@ private static String getMappedColumnValue(
     String colInputValue = "";
     Type colType = spannerColDef.type();
     String colName = spannerColDef.name();
-    if (colType.getCode().equals(Type.Code.FLOAT64)) {
+    if (colType.getCode().equals(Type.Code.FLOAT64)
+        || colType.getCode().equals(Type.Code.FLOAT32)
+        || colType.getCode().equals(Type.Code.PG_FLOAT4)) {
```

Comment:
I think it would be better if we add support for all PG types together

Reply:
Makes sense. Removed the PG part.